public/config.js

```diff
@@ -0,0 +1,5 @@
+var MAIN_CONFIG =  {
```

Comment:
This file must not be part of the repo. But information about this must be in REDME.